plugins/frontend/backstage-plugin-prometheus/src/types.ts

```diff
@@ -79,3 +79,5 @@ export type PrometheusRuleResponse = {
   };
   status: 'success' | 'error';
 };
+
+export type OnRowClick = (arg: PrometheusDisplayableAlert | undefined) => void;
```

Comment:
We'd always have row data in this case right? Otherwise the row itself would be visible/clickable at all.

Reply:
Fixed thank you :+1: